docs/source/reference/databases/hackernews.rst

```diff
@@ -0,0 +1,44 @@
+Hackernews
+==========
+
+The connection to Hackernews is based on the `Algolia Hackernews <https://hn.algolia.com/api>`_ API.
+
+Dependency
+----------
+
+* requests
+
+
+Parameters
+----------
+
+Required:
+
+* ``query`` is the search query for getting the results.
+
+Optional:
+
+* ``tags`` is the tag used for filtering the query results. Check `available tags <https://hn.algolia.com/api#:~:text=filter%20on%20a%20specific%20tag.%20Available%20tags%3A>`_ to see a list of available filter tags.
+
+Create Connection
+-----------------
+
+.. code-block:: text
+
+   CREATE DATABASE hackernews_data WITH ENGINE = 'hackernews', PARAMETERS = {
+        "owner": "EVADB",
+        "repo": "story"
```

Comment:
I think it should be query and tag instead of owner and repo.

Reply:
Thanks! fixed it.